src/postprocess/elastic.jl

```diff
@@ -0,0 +1,59 @@
+import DifferentiationInterface as DI
+
+
+function symmetries_from_strain(model0, voigt_strain)
+    lattice = DFTK.voigt_strain_to_full(voigt_strain) * model0.lattice
+    model = Model(model0; lattice, symmetries=true)
+    model.symmetries
+end
+
+function stress_from_strain(model0, voigt_strain; symmetries, Ecut, kgrid, tol, kwargs...)
+    # TODO restart SCF from previous
+    lattice = DFTK.voigt_strain_to_full(voigt_strain) * model0.lattice
+    model = Model(model0; lattice, symmetries)
+    basis = PlaneWaveBasis(model; Ecut, kgrid)
+    scfres = self_consistent_field(basis; tol, kwargs...)
+    DFTK.full_stress_to_voigt(compute_stresses_cart(scfres))
+end
+
+"""Computes the *clamped-ion* elastic constants (without ionic relaxation)"""
+function elastic_constants(scfres::NamedTuple; 
```

Comment:
How about `elastic_tensor` because that's what it is. The constants are the individual entries.